creed/file_hash_test.py

```diff
@@ -0,0 +1,56 @@
+import os
+import shutil
+import tempfile
+import pytest
+
+import skimage.io as io
+import numpy as np
+
+from creed import file_hash
+
+
+def test_get_hash():
```

Comment:
You can test the `==` case too by using `shutil.copyfile()` to duplicate `test_file_0.tiff` into `test_file_1.tiff`.